include/deal.II/non_matching/mapping_info.h

```diff
@@ -207,6 +207,19 @@ namespace NonMatching
     using VectorizedArrayType = typename dealii::internal::VectorizedArrayTrait<
       Number>::vectorized_value_type;
 
+    /**
+     * Collects the options which can be used to specify the behavior during
+     * reinitialization.
+     */
+    struct AdditionalData
+    {
+      /**
+       * During reinitialization Quatratures with global weights are given
+       * as, e.g., obtained by QSimplex::compute_affine_transformation().
+       */
+      bool use_global_weights = false;
```

Comment:
Could you intialize this in the constructor? This is how do it in the other instances.

Reply:
Done, as suggested :)